cmd/skycoin/skycoin.go

```diff
@@ -295,7 +101,7 @@ var devConfig = Config{
 
 	LaunchBrowser: false,
 	// Data directory holds app data -- defaults to ~/.skycoin
-	DataDirectory: filepath.Join(home, ".skycoin"),
+	DataDirectory: "~/.skycoin",
```

Comment:
I think we should use `$HOME` and manually substitute that

Reply:
yea right now I am substituting `~` with the home path.